### DISCUSSION TITLE
MySQL 数据库列表中支持添加自己创建的数据库吗

### DISCUSSION THREAD

**Discussion by e8-LiuPeng** [1Panel-dev/1Panel#4946] [Q&A] [answered]
在这里，支持添加自己创建的数据库吗
![image](https://github.com/1Panel-dev/1Panel/assets/121592444/034934a9-b93e-4cd7-ad97-8b7bb8efd4e3)

**Comment by wan92hen** [comment]
可以先点击远程服务器按钮连接远程数据库

**Comment by e8-LiuPeng** [comment, marked as answer]
找到如何解决了：可以直接点击 **从服务器同步** 按钮。
